tests/utils/revert_language_settings.py

```diff
@@ -4,16 +4,15 @@
 
 
 def revert_language_settings(func):
-    def wrapper(language_alias, schema, db_conn,
-                container_path, language_settings):
+    def wrapper(**kwargs):
         try:
-            return func(language_alias, schema, db_conn,
-                        container_path, language_settings)
+            return func(**kwargs)
         except Exception as exc:
             print("Exception occurred while running the test: %s" % exc)
             raise pytest.fail(exc)
         finally:
             print("Revert language settings")
```

Comment:
generally I only would use prints in the main function, within library code I would suggest the use of logging.

Reply:
thanks